enclave_wrangler/datasets.py

```diff
@@ -342,21 +342,27 @@ def transform(fav: dict) -> pd.DataFrame:
     return df
 
 
-def get_last_vocab_update():
-    """
-    https://unite.nih.gov/workspace/documentation/developer/api/catalog/services/CatalogService/endpoints/getTransaction
-    https://unite.nih.gov/workspace/documentation/developer/api/catalog/objects/com.palantir.foundry.catalog.api.transactions.Transaction
-    """
-    dataset_rid = FAVORITE_DATASETS['concept']['rid']
+def get_last_update_of_datasets(rid: str):
```

Comment:
I simplified `get_last_vocab_update ()` and abstracted logic from it into new `get_last_update_of_dataset()`, which I will also be using to check when counts tables were updated, as I assume they _might_ not be updated at the same time.

Reply:
### Vocab and counts update time differs
```py
    x1 = get_last_update_of_dataset('concept_set_counts_clamped')  # '2023-08-03T23:46:00.951Z'
    x2 = get_last_update_of_dataset('deidentified_term_usage_by_domain_clamped')  # '2023-08-03T22:31:09.607Z'
    x3 = get_last_vocab_update()  # '2023-08-05T15:15:52.706Z
```
Q1: Do counts tables get updated at roughly the same time than vocab tables?
No. The last time the counts were updated, they were updated 2 days earlier, or rather, I would assume 5 days after the vocab tables were previously updated.
Q2: Does each count table get updated at roughly the same time?
Yes. The timestamps differ by about an hour and a half, but that is not so significant. I imagine they are updated via the same scheduled pipeline.

Reply:
@Sigfried Also, what do you think that "clamped" means?